File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -382,7 +340,7 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Hyperlinks
 
-The Blink HTML converter support preserving URL links from HTML to PDF. Refer to the following code snippet.
+The Blink HTML converter support preserving URL links from HTML to PDF using [EnableHyperLink ](https://help.syncfusion.com/cr/file-formats/Syncfusion.HtmlConverter.BlinkConverterSettings.html#Syncfusion_HtmlConverter_BlinkConverterSettings_EnableHyperLink) property. Refer to the following code snippet.
```

Comment:
using [EnableHyperLink]  - Remove the space 
using [EnableHyperLink]() property in [BlinkConverterSettings]() class

Reply:
Modified